code/global/Utils.cs

```diff
@@ -70,6 +70,21 @@ public static List<TTTPlayer> GetConfirmedPlayers()
             return players;
         }
 
+        public static IEnumerable<Client> GetClientsSpectatingPlayerAndPlayer(TTTPlayer player)
```

Comment:
I'd remove `AndPlayer` as the player itself more or less observes the own client